cfme/tests/services/test_service_performance.py

```diff
@@ -0,0 +1,38 @@
+from timeit import timeit
+
+import pytest
+
+from cfme import test_requirements
+from cfme.base.ui import navigate_to
+from cfme.services.myservice import MyService
+from cfme.tests.test_db_migrate import download_and_migrate_db
+from cfme.utils.conf import cfme_data
+
+
+@pytest.fixture
+def appliance_with_performance_db(temp_appliance_extended_db):
+    app = temp_appliance_extended_db
+    db_backups = cfme_data['db_backups']
+    performance_db = db_backups['performance_510']
```

Comment:
Should this key be parametrized? Does it depend on the appliance version under test?

Reply:
no. the DB with many services pre-created is migrated to the required version